examples/units_and_coordinates/stereo_loc_solarmach.py

```diff
@@ -0,0 +1,53 @@
+"""
+===================================
+Getting the location of STEREO-A using solarmach
+===================================
+
+How to get the position of planetary bodies im the solar system using
+`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
```

Comment:
```suggestion
This example showcases how to get the position the planetary bodies within the solar system using `solarmach <https://github.com/jgieseler/solarmach/>`__.
```
This text was for the other example.